src/libcommonserver/utility/utility.cpp

```diff
@@ -913,4 +913,9 @@ std::string Utility::userName() {
     return userName_private();
 }
 
+ExitInfo Utility::terminateThreadFunction(ExitInfo exitInfo) {
```

Comment:
So this method can be `void` now?

Reply:
Doesn't exist anymore after the merge of "Kdesktop 1313 memory leaks"